vaadin-multi-select-list-mixin.html

```diff
@@ -73,6 +76,10 @@
       }
     }
 
+    /**
+     * @param {MouseEvent} event
```

Comment:
```suggestion
     * @param {MouseEvent!} event
```
(avoid `|null` in result)

Reply:
Done.